src/HucaresServer.Storage/Helpers/MissingPlateHelper.cs

```diff
@@ -43,6 +43,14 @@ public IEnumerable<MissingLicensePlate> GetAllPlateRecords()
             }
         }
 
+        public MissingLicensePlate GetPlateRecordByPlateNumber(string plateNumber)
+        {
+            using (var ctx = _dbContextFactory.BuildHucaresContext())
+            {
+                return ctx.MissingLicensePlates.FirstOrDefault(c => c.PlateNumber == plateNumber);
```

Comment:
This should return all the records with the plate number, not just the first one. Change tests accordingly.